matrix.py

```diff
@@ -0,0 +1,156 @@
+            print("cannot subtract from a / a scalar")
+
+    def __mul__(self, other):
+        """Multiplies matrix by scalar/matrix."""
+
+        if isinstance(other, Matrix):
+            if len(self.data) == len(other.data):
+                # Converting the second matrix to columns
+                column_matrix = [tup[index] for index in range(len(self.data))
+                                 for tup in other.data]
+                split_column_matrix = splitting(column_matrix, len(self.data))
+                post_mul = []
+                for tup in self.data:
+                    for split_list in split_column_matrix:
+                        for num, index in \
+                                zip(split_list, range(len(self.data))):
+                            post_mul.append(num * tup[index])
+                split_mul = splitting(post_mul,
+                                      len(self.data) * len(self.data))
+                added = [sum(tuples) for tuples in split_mul]
+                final_mtx = splitting(added, len(self.data))
+                return Matrix(tuple(final_mtx))
+            else:
+                print("cannot multiply different dimension matrices")
+
+        else:
+            final_mtx = [num * other for tup in self.data for num in tup]
+            return Matrix(tuple(splitting(final_mtx, len(self.data))))
+
+    def __truediv__(self, other):
+        """Divide matrix by scalar."""
+
+        if not isinstance(other, Matrix):
+            divided_mtx = [num / other for tup in self.data for num in tup]
+            return Matrix(tuple(splitting(divided_mtx, len(self.data))))
+
+        else:
+            print("cannot divide matrix by matrix")
+
+    @staticmethod
+    def unity(num):
+        """Display tuple of tuples with 1 & 0
+        so that the ones create a diagonal if seen as a square.
+
+            Args:
+                num: the amount of ones the user wants
+
+            Returns: the final tuple, as a Matrix type
+        """
+
+        unity_list = [0 for _ in range(num * num)]
+        unity_list[::num + 1] = [1 for _ in unity_list[::num + 1]]
+        return Matrix(tuple(splitting(unity_list, num)))
+
+    @staticmethod
+    def ones(num):
+        """Display a tuple of tuples filled with ones.
+
+            Args:
+                num: the amount of tuples the user wants
+
+            Returns: the final tuple, as a Matrix type
+        """
+
+        ones_list = [1 for _ in range(num * num)]
+        return Matrix(tuple(splitting(ones_list, num)))
+
+    @property
+    def tuples(self):
+        """Display the instance as tuple.
+
+            Returns: the raw tuple
+        """
+
+        return self.data
+
+    # END OF CLASS --------------
+
+
+def splitting(a_list, slice_number):
+    """Split a list into smaller lists and convert them to tuples.
+
+        Args:
+            a_list: the original unsliced list
+            slice_number: the amount of small lists post slicing
```

Comment:
specify types for all arguments that you get in all your functions in all docs

```suggestion
            a_list (list): the original unsliced list
            slice_number (number): the amount of small lists post slicing
```